docs/src/pages/demos/app-bar/BottomAppBar.js

```diff
@@ -111,7 +112,9 @@ function BottomAppBar() {
               {id === 1 && <ListSubheader className={classes.subheader}>Today</ListSubheader>}
               {id === 3 && <ListSubheader className={classes.subheader}>Yesterday</ListSubheader>}
               <ListItem button>
-                <Avatar alt="Profile Picture" src={person} />
+                <ListItemAvatar>
```

Comment:
Is it still valid to use `Avatar` without the `ListItemAvatar` or should we include this in the breaking changes?

Reply:
We have `The usage of the ListItemAvatar component is required when using an avatar` listed in the breaking change. Does it work for you?

Reply:
Sure.